doc/faq.mkd

```diff
@@ -1,6 +1,41 @@
 Frequently Asked Questions
 ==========================
 
+### How can run `puppet generate types` for each changed environment on deployment?
```

Comment:
This heading doesn't read too well.  Is it just missing an `I`? (`How can I run...`)

Reply:
Good catch. I definitely meant to put the `I` in there. Though I'm not sure that's the best either, do you have any preferences for other ways to format this heading?